zipkin-server/src/main/java/zipkin/server/internal/MetricsHealthController.java

```diff
@@ -43,4 +59,31 @@ public ObjectNode fetchMetricsFromMicrometer(){
     }
     return metrics;
   }
+
+  // Delegates the health endpoint from the Actuator to the root context path and can be deprecated
+  // in future in favour of Actuator endpoints
+  @GetMapping("/health")
+  public Map getHealth() {
+    Map health = new HashMap();
+    health.put("status", healthEndpointDelegate.health().getStatus().getCode());
+    health.put("zipkin", healthEndpointDelegate.health().getDetails().get("zipkin"));
+    return health;
+  }
+
+  // Prometheus scrape code ported from springboot actuator and can be deprecated later
+  // through root context path for backward compatibility, Mimics the same code from
+  // org.springframework.boot.actuate.metrics.export.prometheus.PrometheusScrapeEndpoint
+  @GetMapping("/prometheus")
```

Comment:
If this endpoint does the same thing as the Actuator version, we could just redirect or forward that route. This is what the Spring Initializr did to maintain backwards compatibility with its `/info` endpoint: https://github.com/spring-io/initializr/blob/540918567b60da18341883750a5a327f4456fd84/initializr-web/src/main/java/io/spring/initializr/web/autoconfigure/InitializrWebConfig.java#L47

Reply:
Makes sense. I will change it.